tensorflow/python/ops/nn_ops.py

```diff
@@ -42,6 +43,96 @@
 local_response_normalization = gen_nn_ops.lrn
 
 
+def confusion_matrix(predictions, targets, name=None):
+  """'
+  Calculate the Confusion Matrix for a pair of prediction and target 1-D int arrays.
```

Comment:
Put on one line:

"""Computes the confusion matrix from predictions and targets.